packages/js/manifests/polywrap/src/formats/polywrap.test/deserialize.ts

```diff
@@ -46,8 +46,11 @@ export function deserializePolywrapWorkflow(
   );
 
   if (versionCompare === -1) {
+    // Warn user to migrate their manifest
+    options?.logger?.warn(`PolywrapWorkflow is using an older version of the manifest format (${anyPolywrapWorkflow.format}). Please update your manifest to the latest version (${latestPolywrapWorkflowFormat}) by using the "polywrap manifest migrate <type>" command.`);
```

Comment:
This looks fine to me, but just want to call out that we're making an explicit reference here to the Polywrap CLI, which is an external dependency to this package.